frontend/ui/impl/src/test/resources/expected-error-report-uris/for_multiple_events.txt

```diff
@@ -0,0 +1,52 @@
+https://github.com/VirtusLab/git-machete-intellij-plugin/issues/new
+?title=java.lang.RuntimeException%3A+exception+message%3B+java.lang.RuntimeException%3A+another+exception+message
+&labels=bug
+&body=%23%23+Running+environment%0A
+-+Git+Machete+plugin+version+-+mock+plugin+version%0A
+-+IDE+-+mocked+IntelliJ+idea%0A
+-+Operating+system+-+Mock+OS+X+Hehe%0A
+%0A
+%23%23+IDEA+-+stack+trace%0A
+%60%60%60%0A
+java.lang.RuntimeException%3A+exception+message%0A
```

Comment:
The case of multiple exceptions being reported at the same time (since _nieszczęścia chodzą parami_)